source/_docs/migrate-wordpress.md

```diff
@@ -64,6 +64,9 @@ Click your browser's back button from the Pantheon Dashboard and re-authenticate
 
 ### Import Failed
 This error can occur on sites using a content delivery network (CDN) service that is not configured to allow the POST HTTP method. Resolve this issue by [temporarily setting POST as an allowed HTTP method within the CDN's configuration](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/distribution-web-values-specify.html#DownloadDistValuesAllowedHTTPMethods) and restart the migration process. Once the site has been successfully migrated, the POST HTTP method can be disabled.
+
+Imports can also fail for very large sites, which may time out while importing. In these cases, initiate the migration again from the source site, and the transfer should pick up where it left off.
```

Comment:
What do you think about having subheads under Import Failed?
e.g.  CDN blocking POST requests, Very large site footprint